testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -418,6 +419,48 @@ QUnit.module('typing', moduleConfig, () => {
         assert.equal(handler.callCount, 2, '\'change\' event is not fired after focus out without value change');
         clock.restore();
     });
+
+    QUnit.test('TextEditor with mask option should work correctly with autofill (T869537)', function(assert) {
```

Comment:
This test looks very complex. Can we refactor it (split to several, parametrize, get rid of unnecessary `if`, etc)?